syncr_backend/network/listen_requests.py

```diff
@@ -38,13 +40,34 @@ def request_dispatcher(request: dict, conn: socket.socket) -> None:
         REQUEST_TYPE_CHUNK_LIST: handle_request_chunk_list,
         REQUEST_TYPE_CHUNK: handle_request_chunk,
         REQUEST_TYPE_NEW_DROP_METADATA: handle_request_new_drop_metadata,
+        REQUEST_TYPE_FRONTEND_MESSAGE: handle_request_frontend_message,
```

Comment:
We probably don't want to handle this on the same socket.  We should set up a new one.  Probably just delete the changes to this file, and we'll work that out later.

Reply:
By this do you mean deleting all changes in listen_requests (i.e. deleting handle_request_frontend_message function, etc.)?

Reply:
delete changes to this file, I want to use a different socket for frontend<->backend communication